src/main/java/stsjorbsmod/powers/SacrificePower.java

```diff
@@ -0,0 +1,63 @@
+package stsjorbsmod.powers;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.GainEnergyAction;
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.actions.utility.UseCardAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import stsjorbsmod.actions.DecreaseMaxHpAction;
+
+public class SacrificePower extends CustomJorbsModPower {
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(SacrificePower.class);
+    public static final String POWER_ID = STATIC.ID;
+
+    public SacrificePower(AbstractPlayer owner, int stacks) {
+        super(STATIC);
+        this.isTurnBased = true;
+        this.amount = stacks;
+        this.owner = owner;
+        this.updateDescription();
+    }
+
+    @Override
+    public void onUseCard(AbstractCard card, UseCardAction action) {
+        super.onUseCard(card, action);
+
+        int hpCost = 0;
+        if (card.costForTurn != -1) {
+            hpCost = card.costForTurn;
+            card.freeToPlayOnce = true;
+        }
+        else {
+            hpCost = AbstractDungeon.player.energy.energy;
+            addToBot(new GainEnergyAction(hpCost)); // Makes X cost cards "free"
```

Comment:
I would have thought you'd be able to use `card.freeToPlayOnce` instead of this for X cost cards, based on looking over a handful of the base game cases; I think that would probably be better for cooperating with other freeToPlayOnce based effects

Reply:
Ah, I see, it's an ordering issue; `AbstractPlayer.useCard` calls `AbstractCard.use` (which reads the `freeToPlayOnce` value) *before* it constructs the corresponding `UseCardAction` (which invokes our power's callback). Hmm.

Reply:
Yeah, I agree though I don't like the implementation...